### PR TITLE
Removed default verbose option

### PR DESCRIPTION
**Pull request by pavankumar526**

### CODE REVIEW

File: sharelib/spark/src/main/java/org.apache.oozie.action.hadoop/SparkMain.java [hunk 1]
```diff
@@ -154,10 +153,6 @@ protected void run(String[] args) throws Exception {
             sparkArgs.add(DIST_FILES + sparkJars);
         }
 
-        if (!sparkArgs.contains(VERBOSE_OPTION)) {
```
Comment (pallavi-rao) [inline sharelib/spark/src/main/java/org.apache.oozie.action.hadoop/SparkMain.java]: Should it rather not be `sparkArgs.contains(VERBOSE_OPTION)` ? If user wants verbose, then you set it?

Comment (pavankumar526) [inline sharelib/spark/src/main/java/org.apache.oozie.action.hadoop/SparkMain.java]: Ya that will be automatically set. User has to just pass verbose option through conf.

Comment (pallavi-rao) [inline sharelib/spark/src/main/java/org.apache.oozie.action.hadoop/SparkMain.java]: Ah! I see. Looks good then.

**Comment by pavankumar526** [comment]
Merging it.

**Pull request merged**